miqi/agent/tools/shell.py

```
@@ -82,7 +82,15 @@ def name(self) -> str:
 
     @property
     def description(self) -> str:
-        return "Execute a shell command and return its output. Use with caution."
+        return (
+            "Execute a shell command in the WSL sandbox and return its output. "
+            "Use with caution. 沙箱工作目录说明：exec 在沙箱中运行，"
```

Ruff (0.16.1)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)